src/Migrations/AwaitPrecedenceMigration.php

```diff
@@ -0,0 +1,71 @@
+<?hh //strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+ namespace Facebook\HHAST\Migrations;
```

Comment:
no leading space; hackfmt (or in-IDE code formatting) should fix this

Reply:
I've got a bunch more whitespace nits, but hackfmt should fix them all.Let me know if you want to fcomment them, but seems like it will just be noise